src/driver/driver_api.cc

```diff
@@ -34,6 +34,8 @@
 #include <mutex>
 #include <stack>
 
+#include "../relay/backend/te_compiler.h"
```

Comment:
I'm not sure it is needed in this file.

Reply:
Ah, good spot, this was when I was trying to bundle building a unified `IRModule` into this PR - I'll fix :smile_cat: